alefnsc/metamask-mobile
# Experimental/in app logo replacement

## **Description**

This logo replacement has been requested by Mahindra Majumdar as part of an initial screening of GPTCore hiring process.

## **Related issues**
 
Fix: There is a minor patch in the code which was preventing the application build.
Refactor: Included the Chainlink name and logo into the project and renamed variables and paths to consider these new uploaded images.

## **Manual testing steps**

1. Run the application locally.
2.  Navigate to the welcome screen.
3. The application name in the welcome screen at the top expect to be Chainlink.
4. Navigate to the home screen.
5.  The application icon on the home screen at the top left corner expect to be a Hexagon.

## **Screenshots/Recordings**

- Welcome Screen

<img width="312" alt="Welcome Screen" src="https://github.com/alefnsc/metamask-mobile/assets/101351170/51d2acc0-ddd2-49d5-b316-31aa86dac768">

- Home Screen

<img width="336" alt="Home Screen" src="https://github.com/alefnsc/metamask-mobile/assets/101351170/ec26c8bc-590b-4c52-9726-654d3b7983d1">

- Unlock Screen

<img width="475" alt="Unlock Screen" src="https://github.com/alefnsc/metamask-mobile/assets/101351170/ddd6b96b-733c-47c7-b80b-ac8fd30f1378">
